src/views/Home/Home.test.jsx

```diff
@@ -10,6 +12,17 @@ const user = {
   color: 'crimson',
 }
 
-test('Should render the user profile', () => {
-
+test('Should render the user profile', async () => {
+  render(
+  <Home user={user} />
+  )
+  // screen.getByText(/loading/i);
+  await screen.findByAltText('header');
+  screen.getByText(/vonta/i);
+  screen.getByText(/verba/i);
+  screen.getByText(/interests/i);
+  screen.getByAltText('avatar');
+  screen.getByAltText('header');
+  screen.getAllByRole('listitem');
```

Comment:
Don't forget to add in variable names in the future so we can use our expect statement to make the test complete. Also, for our getAllByRole('listitem'), we will want to expect('listitem'.children.length).toEqual(user.likes.length) to make the test more specific.